vernac/vernacinterp.ml

```diff
@@ -206,13 +206,13 @@ and vernac_load ~verbosely fname =
     CErrors.user_err Pp.(str "Files processed by Load cannot leave open proofs.");
   stack, pm
 
-and interp_control ~st ({ CAst.v = cmd } as vernac) =
+and interp_control ~st ({ CAst.v = cmd; loc } as vernac) =
```

Comment:
This is the loc of the entire vernacular command, later on one can only see `cmd.expr`.
So, as we do with `~attrs`, we pass the `~loc` along

Reply:
Yeah indeed it seems we should modify the AST so the vernac gets a `CAst` node too, but that could be done later on [maybe add a comment to the effect]